Add rest apis to new_restructure (#2385)
wso2/docs-is: en/docs/apis/authentication-data-api.md

These parameters may not be passed in the redirect URL due to one or
many of the following reasons.

-   Sensitivity of the values passed.
-   Complexity of the values passed.
-   Length of the parameters exceeding, or has the possibility of
    exceeding the allowed limits.
-   Compliance to certain business policies.

---

## Configure Authentication Data API

> To make these parameters available via the Authentication Data API, we
> need to configure the Identity Server as follows.

Reviewer: ```suggestion
To make these parameters available via the Authentication Data API, apply the following configurations. 
```
